.github/workflows/submit-to-obs.yml

```
@@ -0,0 +1,74 @@
+name: Rebuild trento in obs
+on: [push]
+env:
+  PACKAGE_NAME: trento
+  OBS_USER: ${{ secrets.OBS_USER }}
+  OBS_PASS: ${{ secrets.OBS_PASS }}
+  OBS_PROJECT: ${{ secrets.OBS_PROJECT}}
+  TARGET_PROJECT: ${{ secrets.TARGET_PROJECT}}
+  FOLDER: packaging/suse
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+  delivery:
+    needs: validation
+    runs-on: ubuntu-18.04
+    if: ${{ github.event_name != 'pull_request' }}
```

this should actually only happen on the main branch:
```suggestion
    if: github.ref == 'refs/heads/main'
```

---

:+1: